website/content/docs/updates/important-changes.mdx

```diff
@@ -147,6 +147,27 @@ None.
 
 ## Known issues
 
+### UI login fails for auth mounts with underscores and unauthenticated listing ((#ui-login-underscore))
+
+| Change       | Affected version | Fixed version
+| ------------ | ---------------- | --------------------
+| Known issue  | 1.20.0           | 1.20.1
+
+If an auth method is mounted at a path that includes an underscore (e.g. oidc_test)
+**and** the mount is tuned with `listing_visibility="unauth"` the login request will
+fail because the UI requests the wrong endpoint. Mounts with dashes, e.g. oidc-test
+still log in fine.
+
+#### Recommendation
+
+As a workaround, you can log in to the UI using the following steps:
+1. Navigate directly to the URL for that method’s type `${VAULT_ADDR}/ui/vault/auth?with=oidc`
+2. Click "Sign in with other methods ->"
+3. Select type from the dropdown
```

Comment:
```suggestion
1. Select type from the dropdown
```
Markdown correction: use `1.` for all numbered list items

Question: is "type" a GUI label or do we mean they should select the appropriate type from the dropdown?